index.html

```diff
@@ -359,16 +359,16 @@
               </li>
               <li>If <a>obtaining the manifest</a> results in an error, a user
               agent can, at this point, fall back to using the <a>top-level
-              browsing context</a>' <code>Document</code>'s metadata to
-              populate an <a>installation process</a>' UI.
+              browsing context</a> <code>Document</code>'s metadata to populate
+              an <a>installation process</a> UI.
```

Comment:
I also agree with @mgiuca - @mcgreevy, can you please make the change as @mgiuca suggests above?